### PR TITLE
Added changelog for 3.18.6

### PR DESCRIPTION
**Pull request by larsewi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+	- Added ability to enable backup archives during policy update (ENT-10481)
+	- Added ability to extend without overriding filename patterns to copy
+	  during policy update (ENT-10480)
+	- Added ss to paths for linux (ENT-10413)
+	- Added support for RockyLinux in self upgrade (ENT-10335)
+	- Aligned module build result with release artifact (ENT-10345)
+	- Fixed cfe_autorun_inventory_aws_ec2_metadata_cache file creation
+	  (CFE-4221)
+	- Fixed lib/testing.cf tap output (CFE-4245, CFE-4246)
+	- Guard against /sys/hypervisor/uuid not being readable (ENT-9931)
```
Comment (olehermanse) [inline CHANGELOG.md:13]:
```suggestion
	- Fixed 'regline' error messages which appear on systems (ex. CentOS 6)
	  where /sys/hypervisor/uuid is not readable (ENT-9931)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+	  where /sys/hypervisor/uuid is not readable (ENT-9931)
+	- Made Mission Portal Apache SSLProtocol configurable via augments
+	  (ENT-10412)
+	- Made allowconnects and allowallconnects configurable via Augments
+	  (ENT-10212)
+	- Made lastseenexpireafter in body common control configurable via Augments
+	  (ENT-10414)
+	- Removed considerations for old versions from bundle agent
+	  cfe_autorun_inventory_aws_ec2_metadata_cache (CFE-4222)
+	- Removed jq dependency in lib/testing.cf and fixed tap output (CFE-4242)
```
Comment (olehermanse) [inline CHANGELOG.md:23]:
```suggestion
	- Removed jq dependency in lib/testing.cf and fixed tap output
	   (CFE-4242, CFE-4245, CFE-4246)
```

**Review by olehermanse — APPROVED** [review]

**Comment by larsewi** [comment]
Squashed commits

**Review by olehermanse — APPROVED** [review]

**Review by nickanderson — APPROVED** [review]
I didn't see anything missing from `git log --reverse --no-merges 3.18.5..3.18.x`

**Pull request merged**